src/utils/convert.js

```diff
@@ -117,6 +117,42 @@ const Convert = {
     idxToByteArray: function (input) {
         return converters_1.int32ToBytes(input, true);
     },
+    byteArrayToShort: function (bytes) {
+        if (!(bytes instanceof Array || bytes instanceof Uint8Array)) {
+            throw new Error('Byte array or Uint8Array input is expected');
+        }
+        else if (bytes instanceof Array && !(bytes.every(function (n) { return typeof n === 'number'; }))) {
+            throw new Error('Byte array contains non-numeric elements');
+        }
```

Comment:
This checking appears many times. Extract to a function. If input is Invalid, just throw error.